scripts/release.sh

```diff
@@ -184,7 +184,7 @@ main() {
     fi
 
     # Tag release.
-    if [ "$(git tag --list | grep -c "${RELEASE_VERSION}")" -gt 0 ]; then
+    if git tag --list | grep --quiet "^${RELEASE_VERSION}$"; then
```

Comment:
After second thought, one benefit of the previous implementation is that it can prevent from creating a tag lower than any of existing available tags. For example, assuming 3.5.99 already exists, then we can release 3.5.23.

The problem for now is that it regards v3.6.0 < v3.6.0-rc.x.

The fix is working for now, it's up to the release team how to improve it in future.

Thanks @joshjms